Mathlib/Topology/Baire/CompleteMetrizable.lean

```diff
@@ -43,17 +43,17 @@ instance (priority := 100) BaireSpace.of_pseudoEMetricSpace_completeSpace : Bair
     obtain ⟨r, rpos, hr⟩ : ∃ r > 0, closedBall y r ⊆ f n :=
       nhds_basis_closed_eball.mem_iff.1 (isOpen_iff_mem_nhds.1 (ho n) y ys)
     refine' ⟨y, min (min (δ / 2) r) (B (n + 1)), _, _, fun z hz => ⟨_, _⟩⟩
-    show 0 < min (min (δ / 2) r) (B (n + 1))
-    exact lt_min (lt_min (ENNReal.half_pos δpos) rpos) (Bpos (n + 1))
-    show min (min (δ / 2) r) (B (n + 1)) ≤ B (n + 1)
-    exact min_le_right _ _
-    show z ∈ closedBall x δ
-    exact
-      calc
-        edist z x ≤ edist z y + edist y x := edist_triangle _ _ _
-        _ ≤ min (min (δ / 2) r) (B (n + 1)) + δ / 2 := add_le_add hz (le_of_lt xy)
-        _ ≤ δ / 2 + δ / 2 := (add_le_add (le_trans (min_le_left _ _) (min_le_left _ _)) le_rfl)
-        _ = δ := ENNReal.add_halves δ
+    · show 0 < min (min (δ / 2) r) (B (n + 1))
+      exact lt_min (lt_min (ENNReal.half_pos δpos) rpos) (Bpos (n + 1))
+    · show min (min (δ / 2) r) (B (n + 1)) ≤ B (n + 1)
+      exact min_le_right _ _
+    · show z ∈ closedBall x δ
+      exact
+        calc
```

Comment:
`exact calc` is synonymous with `calc` isn't it?

Reply:
I am not sure if it is exactly a synonym, but here it worked just as well.